content/resources.md

```diff
@@ -27,7 +27,7 @@ Also join the @CTF role on Discord to participate in CTF with us!
     - To join, see #announcements or message an officer on Discord.
 - CTF101: [https://ctf101.org/](https://ctf101.org/)
 - TrailOfBits CTF Field Guide: [https://trailofbits.github.io/ctf/](https://trailofbits.github.io/ctf/)
-- CS499: Cyber Attacks And Defense (taught by our awesome advisor, Dr. Jang, check out the Lectures at this link): [https:// cand.unexploitable.systems/cal.html](https:// cand.unexploitable.systems/cal.html)
+- CS499: Cyber Attacks And Defense (taught by our awesome advisor, Dr. Jang, check out the Lectures at this link): [https://cand.unexploitable.systems/cal.html](https://cand.unexploitable.systems/cal.html)
```

Comment:
thanks for this edit.

can you make an issue that we need to update our resources page?